packages/example-codehub/src/codehub-application.ts

```diff
@@ -12,8 +12,9 @@ export class CodeHubApplication extends Application {
     super();
 
     const app = this;
-    const server = new RestServer(this, {});
-    app.server(server);
+    app.server(RestServer);
+    // We can't use the async version in the ctor!
+    const server = app.getServerSync(RestServer);
```

Comment:
Ouch! We cannot rely on sync getter in production code. What if the RestServer is depending on configuration that requires async computation? For example, what if we are fetching the config from Consul?